docs/canvas/canvas-elements.asciidoc

```diff
@@ -16,6 +16,20 @@ When you add elements to your workpad, you can:
 
 * <<element-save,Save your elements>>
 
+[float]
+[[canvas-supported-datasources]]
+=== Supported data sources
+
+By default, every element you add to a workpad uses demo data until you change the data source. The demo data includes a small sample data set that you can use to experiment with your element. When you have finished using the demo data, connect the element to one of the following data sources:
+
+Elasticsearch SQL:: Allows you to access your data in Elasticsearch using SQL syntax. Unfamiliar with writing Elasticsearch SQL queries? For more information, refer to {ref}/sql-spec.html[SQL language].
+
+Elasticsearch raw data:: Allows you to access your raw data in Elasticsearch without the use of aggregations. Use Elasticsearch raw data when you have low volume datasets, or to plot exact, non-aggregated values.
+To use targeted queries, you can enter a query using the Lucene query string syntax. For more information about the Lucene query string sytax, refer to <<lucene-query,Lucene Query Syntax>>.
```

Comment:
Maybe remove the last sentence and make "Lucene query syntax" the link in the previous sentence.